neurokit2/hrv/hrv_nonlinear.py

```diff
@@ -387,6 +420,56 @@ def _hrv_nonlinear_fragmentation(rri, out):
     return out
 
 
+# =============================================================================
+# DFA
+# =============================================================================
+def _hrv_dfa(peaks, rri, out, n_windows="default", **kwargs):
+
+    if 'dfa_windows' in kwargs:
+        dfa_windows = kwargs['dfa_windows']
+    else:
+        dfa_windows = [(4, 11), (12, None)]
+    # Determine max beats
+    if dfa_windows[1][1] is None:
+        max_beats = len(peaks) / 10
+    else:
+        max_beats = dfa_windows[1][1]
+
+    # No. of windows to compute for short and long term
+    if n_windows == "default":
+        n_windows_short = int(dfa_windows[0][1] - dfa_windows[0][0] + 1)
+        n_windows_long = int(max_beats - dfa_windows[1][0] + 1)
+    elif isinstance(n_windows, list):
+        n_windows_short = n_windows[0]
+        n_windows_long = n_windows[1]
+
+    short_window = np.linspace(dfa_windows[0][0], dfa_windows[0][1], n_windows_short).astype(int)
+    long_window = np.linspace(dfa_windows[1][0], int(max_beats), n_windows_long).astype(int)
+
+    # For monofractal
+    out["DFA_alpha1"] = fractal_dfa(rri, multifractal=False, windows=short_window, **kwargs)['slopes'][0]
+    out["DFA_alpha2"] = fractal_dfa(rri, multifractal=False, windows=long_window, **kwargs)['slopes'][0]
+
+    # For multifractal
+    mdfa_alpha1 = fractal_dfa(rri,
+                              multifractal=True,
+                              q=[-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5],
```

Comment:
```suggestion
                              q=range(-5, 5),
```

(note that, in `fractal_dfa()` u need to convert the `range` object to a list

Reply:
I used np.arange(-5, 6) instead